tests/queries/0_stateless/01290_max_execution_speed_distributed.sql

```diff
@@ -21,6 +21,7 @@ FROM system.query_log
 WHERE
     current_database = currentDatabase() AND
     event_date >= yesterday() AND
+    event_time >= now() - INTERVAL '5 MINUTES' AND -- time limit for tests not marked `long` is 3 minutes, 5 should be more than enough
```

Comment:
Maybe we could just reset the settings to their default values. Or just set them only in the query we care about

Reply:
imo we shouldn't read 869678 rows from `system.query_log` anyway, but leave global settings with default values also wouldn't harm 👍🏼